src/api-extension/order.service.ts

```diff
@@ -26,6 +25,73 @@ export class OrderService {
     private readonly productMapper: ProductMapper,
   ) {}
 
+  public async getMetadataForOrder(
+    order: Order,
+    allMetadataSchemaProperties: string[],
+  ) {
+    const standardMetaProperties = allMetadataSchemaProperties.filter(
+      (key) => !key.includes('custom_filed_'),
+    );
+    const customMetaProperties = allMetadataSchemaProperties.filter(
+      (key) => key.length > 13 && key.slice(0, 13) === 'custom_filed_',
+    );
```

Comment:
1. Maybe assign custom_filed_ to const at start of this function or in separate file.
2. Use defined const.length or assign const.length to other variable instead of '13'. It will be better for code reading.